cli bin in node_modules/.bin require is broken

<!--NOTE: -->
<!--- General questions should go to the discord chat instead of the issue tracker.-->

**Describe the bug**

After installing the CLI and running it locally with `./node_modules/.bin/lodestar` you get a module node found error
```
lion@lion-gram:~/Code/eth2.0/test/test-cli-npm$ ./node_modules/.bin/lodestar 
internal/modules/cjs/loader.js:796
    throw err;
    ^

Error: Cannot find module './cmds/dev'
Require stack:
- /home/lion/Code/eth2.0/test/test-cli-npm/node_modules/@chainsafe/lodestar-cli/lib/index.js
- /home/lion/Code/eth2.0/test/test-cli-npm/node_modules/@chainsafe/lodestar-cli/bin/lodestar
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/lion/Code/eth2.0/test/test-cli-npm/node_modules/@chainsafe/lodestar-cli/lib/index.js:5:12)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Module.require (internal/modules/cjs/loader.js:848:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/lion/Code/eth2.0/test/test-cli-npm/node_modules/@chainsafe/lodestar-cli/lib/index.js',
    '/home/lion/Code/eth2.0/test/test-cli-npm/node_modules/@chainsafe/lodestar-cli/bin/lodestar'
  ]
}

```

**Expected behavior**
CLI should work

**Steps to Reproduce**
```
yarn add @chainsafe/lodestar-cli
./node_modules/.bin/lodestar
```

--- a/packages/lodestar-cli/package.json
+++ b/packages/lodestar-cli/package.json
@@ -9,7 +9,10 @@
   },
   "homepage": "https://github.com/ChainSafe/lodestar#readme",
   "main": "lib/index.js",
-  "files": [],
+  "files": [
+    "lib",
+    "bin"
+  ],
   "scripts": {
     "clean": "rm -rf lib && rm -f tsconfig.tsbuildinfo",
     "build": "yarn run build:lib && yarn run build:types",

--- a/packages/lodestar-cli/package.json
+++ b/packages/lodestar-cli/package.json
@@ -10,9 +10,13 @@
   "homepage": "https://github.com/ChainSafe/lodestar#readme",
   "main": "lib/index.js",
   "files": [
-    "lib",
-    "bin"
+    "lib/**/*.d.ts",
+    "lib/**/*.js",
+    "lib/**/*.js.map"
   ],
+  "bin": {
+    "lodestar": "lib/index.js"
+  },
   "scripts": {
     "clean": "rm -rf lib && rm -f tsconfig.tsbuildinfo",
     "build": "yarn run build:lib && yarn run build:types",
@@ -68,9 +72,6 @@
     "rimraf": "^3.0.0",
     "uuidv4": "^6.1.1",
     "yargs": "^15.3.1"
-  },
-  "bin": {
-    "lodestar": "./bin/lodestar"
   },
   "devDependencies": {
     "@types/chai": "4.2.0",

